irc/server.py

```diff
@@ -435,6 +435,25 @@ def handle_dump(self, params):
             for client in channel.clients:
                 print("     ", client.nick, client)
 
+    def handle_ison(self, params):
+        for nick in params.split(" "):
```

Comment:
I think this loop is somewhat misplaced.
For each iteration it will overwrite your response: "303 ...", so it should happen immediatly before the client-check loop ...